api/models.py

```diff
@@ -1,16 +1,42 @@
-from django.core.validators import EmailValidator
 from django.db import models
 from api.logistics.choice_enums import YEAR_IN_SCHOOL_CHOICES, LEARNING_GOALS_CHOICES, EXPERIENTIAL_LEARNING_HOURS_TYPES
 from api.reliability.validators import no_future_dates, hour_instance_validator, minutes_validator, student_id_validator
 
 from auth_backend.modules.common import models as common_models
+from auth_backend.modules.user.models import BaseVologUser
+
+
+class Mentor(models.Model):
+    #I added the foreign key as we have two users, student and mentor. Foreign key is used to distigush which user is which. 
+    """
+    Represents a DAS mentor.
+    """
+    user = models.OneToOneField(BaseVologUser, on_delete=models.CASCADE)
+
+   # pass
+
+
+
+class Student(models.Model):
```

Comment:
Both Student and Mentor should just subclass from the basevologuser class. I did it on my branch but I don't think you merged at that point